client/src/pages/index.tsx

```diff
@@ -14,10 +14,10 @@ const Home: React.FC<{}> = () => {
       maxW="3xl"

```

Comment:
Looks good but still need to change the link in the GitHub button to this repository according to #41